Pages/ProfileHomePage.cs

```diff
@@ -0,0 +1,61 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using QAMars.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QAMars.Pages
+{
+    public class ProfileHomePage : CommonDriver
+    {
+        private readonly IWebDriver driver;
+        public ProfileHomePage(IWebDriver driver)
+        {
+            this.driver = driver;
```

Comment:
you dont need this cause you are doing : CommonDriver

Reply:
I have removed this constructor and relied only on CommonDriver